config/config_wcoss_dell_p3.sh

```diff
@@ -25,3 +25,9 @@ export LMOD_EXTENDED_DEFAULT="yes"
 
 # Load these basic modules
 module load cmake/3.16.2
+
+# Load for MET
+module load python/3.6.3
+export MET_PYTHON=/usrx/local/prod/packages/python/3.6.3
+export MET_PYTHON_CC=-I${MET_PYTHON}/include/python3.6m
+export MET_PYTHON_LD=-L${MET_PYTHON}/lib/\ -lpython3.6m\ -lpthread\ -ldl\ -lutil\ -lm\ -Xlinker\ -export-dynamic
```

Comment:
Is this the best way to describe the linkers etc.
This should be done in the `build_met.sh` script.

Reply:
> Is this the best way to describe the linkers etc.
Not necessarily.

On many machines, we can run "python-config --cflags" to get the value for MET_PYTHON_CC and can run "python-config --ldflags" to get the value for MET_PYTHON_LD, however, some machines, like WCOSS Mars, unfortunately, do not return the path needed for "-L".  For example:
```
[Julie.Prestopnik@m71a3 hpc-stack]$ python-config --ldflags
-lpthread -ldl -lutil -lm -lpython2.7 -Xlinker -export-dynamic
```
That is an important and required value.  So, in this case, for WCOSS, we need to set the path, which is why in this case, it is being done in this way.  Some machines give the correct value and other machines don't even have "python-config" installed.  

> This should be done in the build_met.sh script.
Please see my reply in the previous comment about why I believe this should be done in the config_wcoss_dell_p3.sh file and not in build_met.sh (to summarize, this is specific to WCOSS).